Makefile

```diff
@@ -1,3 +1,7 @@
+ifeq ($(ROOTLESS),1)
```

Comment:
This isn't needed for sideload support PR.

Reply:
its to simplify make package for rootless. I'll remove it if its not needed as well. Going to add localization support in few mins

Reply:
You make a fair point, then again it should be in a separate PR.

Reply:
Got you. Should i separate PRs also for localization support and small UI improvements?

Reply:
Ideally, yeah. Separate.